src/geo/cartodb-layer-group.js

```diff
@@ -63,14 +67,21 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     });
   },
 
-  _replaceLayerIndexesOfVisibleLayers: function (url) {
-    var remoteIndexesOfVisibleLayers = _.reduce(this.getLayers(), function (indexes, layerModel, layerIndex) {
-      if (layerModel.isVisible()) {
-        indexes.push(this.get('indexOfLayersInWindshaft')[layerIndex]);
-      }
-      return indexes;
-    }.bind(this), []);
-    return url.replace('{layerIndexes}', remoteIndexesOfVisibleLayers.join(','));
+  _replaceLayerIndexesOfVisibleLayers: function (type, url) {
+    if (type === 'png') {
+      var remoteIndexesOfVisibleLayers = _.reduce(this.getLayers(), function (indexes, layerModel, layerIndex) {
+        if (layerModel.isVisible()) {
+          indexes.push(this.get('indexOfLayersInWindshaft')[layerIndex]);
+        }
+        return indexes;
+      }.bind(this), []);
+
+      return url.replace('{layerIndexes}', remoteIndexesOfVisibleLayers.join(','));
+    } else if (type === 'mvt') {
+      return url
+        .replace('{layerIndexes}', 'mapnik')
```

Comment:
Raster tiles have an index of the layer if a map has multiples of them... MVT doesn't, all of them go in the protobufer and they are parsed by tangram.